### PR TITLE
uniq added under UUID

### PR DESCRIPTION
**Pull request by robmuh**
- github.com repo: https://gitlab.com/skilstak/code/uniq/go"
- godoc.org: https://godoc.org/gitlab.com/skilstak/code/uniq/go
- goreportcard.com: https://goreportcard.com/report/gitlab.com/skilstak/code/uniq/go
- coverage service link: https://gocover.io/gitlab.com/skilstak/code/uniq/go

- [x ] I have added my package in alphabetical order.
- [x ] I have an appropriate description with correct grammar.
- [x ] I know that this package was not listed before.
- [x ] I have added godoc link to the repo and to my pull request.
- [x ] I have added coverage service link to the repo and to my pull request.
- [x ] I have added goreportcard link to the repo and to my pull request.
- [x ] I have read [Contribution guidelines](https://github.com/avelino/awesome-go/blob/master/CONTRIBUTING.md#contribution-guidelines), [maintainers note](https://github.com/avelino/awesome-go/blob/master/CONTRIBUTING.md#maintainers) and [Quality standard](https://github.com/avelino/awesome-go/blob/master/CONTRIBUTING.md#quality-standard).

You're awesome too!

### CODE REVIEW

**Comment by robmuh** [comment]
This appears to be a bug or limitation in the `go vet` tool itself probably because of `go test` using reflection and `go vet` does not. While I've been unable to find written proof, I'm quite sure `go vet` is not meant to be run on unit test files, which is why it avoids them when simply run as documented without any argument.

The naming of the example functions in `uniq_test.go` strictly follow the [ well documented]( https://golang.org/pkg/testing/) standard and the tests are clearly being run with `go test`. Below I have added an extra output line to force a failure. Later is the the same removed and passing.

```
$ go test
--- FAIL: ExampleBytes (0.00s)
got:
8
something
want:
8
FAIL
exit status 1
FAIL    gitlab.com/skilstak/code/uniq/go        0.001s
```

After removing `fmt.Println("something")`:

```
PASS
ok      gitlab.com/skilstak/code/uniq/go        0.001s
```

**Comment by robmuh** [comment]
After hours of searching I have found that this is indeed a bug with `go vet`. Here's the thread. https://prod.liveshare.vsengsaas.visualstudio.com/join?3575ADFE711EF9FD4F6A86331671475C7DDB

File: README.md [hunk 1]
```diff
@@ -242,10 +242,12 @@ Please take a quick gander at the [contribution guidelines](https://github.com/a
...
 * [tabby](https://github.com/cheynewallace/tabby) - A tiny library for super simple Golang tables.
 * [tabular](https://github.com/InVisionApp/tabular) - Print ASCII tables from command line utilities without the need to pass large sets of data to the API.
+* [tcell](https://github.com/gdamore/tcell) - Provides a cell based view for text terminals, like xterm. 
 * [termbox-go](https://github.com/nsf/termbox-go) - Termbox is a library for creating cross-platform text-based interfaces.
 * [termdash](https://github.com/mum4k/termdash) - Go terminal dashboard based on **termbox-go** and inspired by [termui](https://github.com/gizak/termui).
 * [termtables](https://github.com/apcera/termtables) - Go port of the Ruby library [terminal-tables](https://github.com/tj/terminal-table) for simple ASCII table generation as well as providing markdown and HTML output.
 * [termui](https://github.com/gizak/termui) - Go terminal dashboard based on **termbox-go** and inspired by [blessed-contrib](https://github.com/yaronn/blessed-contrib).
+* [tview](https://github.com/rivo/tview) - Provides commonly needed components for terminal based user interfaces.
```
Comment (avelino) [inline README.md]: add more packages on this PR? update all links pls

Comment (robmuh) [reply README.md]: Well those are not my packages but happy to add the links for them. I did notice these were recommended by another (who was closed with a "send PR request please") so I thought I would add them. Both of these have the links on their README.md files so just a matter of copying them over.

## Tview Links

* github.com repo: https://github.com/rivo/tview
* godoc.org: https://godoc.org/github.com/rivo/tview
* goreportcard.com: https://goreportcard.com/report/github.com/rivo/tview
* coverage service link: https://gocover.io/github.com/rivo/tview

## Tcell Links

* github.com repo: https://github.com/gdamore/tcell
* godoc.org: https://godoc.org/github.com/gdamore/tcell
* goreportcard.com: https://goreportcard.com/report/github.com/gdamore/tcell
* coverage service link:  https://gocover.io/github.com/gdamore/tcell

Looks like both of them have some work to do to get up to par. I suppose we can leave this pull request open until they score better so people can know they need some help over there to polish them up. I'll try to get to it, but if not anyone can jump in and help them so they qualify to get posted.

**Comment by robmuh** [comment]
Alright I pulled the tview and tcell. I'll submit them as another PR.

**Pull request merged**